lapce-app/src/file_explorer/data.rs

```diff
@@ -399,6 +401,58 @@ impl FileExplorerData {
         }
     }
 
+    pub fn select(&self, path: PathBuf) {
```

Comment:
this should be `reveal` or something about presenting it in view, `select` has specific meaning in UI

Reply:
done